docs/sources/panels-visualizations/_index.md

```diff
@@ -16,11 +16,18 @@ keywords:
 
 # Panels and visualizations
 
-The _panel_ is the basic visualization building block in Grafana. Each panel has a query editor specific to the data source selected in the panel. The query editor allows you to extract the perfect visualization to display on the panel.
+The _panel_ is the basic visualization building block in Grafana.
+Each panel has a query editor specific to the data source selected in the panel.
+The query editor allows you to extract the perfect visualization to display on the panel.
 
-There are a wide variety of styling and formatting options for each panel. Panels can be dragged and dropped and rearranged on the dashboard. They can also be resized.
+There are a wide variety of styling and formatting options for each panel.
+Panels can be dragged, dropped, and resized to rearrange them on the dashboard.
 
-Before you begin, ensure that you have configured a data source. For more information about data sources, refer to [Data Sources]({{< relref "../administration/data-source-management/" >}}).
+Before you add a panel, ensure that you have configured a data source.
+While you can use the [special built-in "Grafana" data source]({{< relref "../datasources/#special-data-sources" >}}) to generate random data for testing, you can accomplish more with panels once you add a live data source.
```

Comment:
```suggestion
```

Not sure we need to mention the built in test data source and random data testing queries here